README.md

```diff
@@ -48,6 +48,10 @@ if (isDevelopment) {
       '@artsy/reaction',
       '@artsy/artsy-xapp'
     ]
+
+    // Set this value to true in order to clear the cache for a module and along
+    // with its tree of dependencies.
+    recursive: false
```

Comment:
Nice. Can you expand the comment a bit so that its clear what is happening when `false`?

Reply:
Sure, I'll elaborate